pkg/metrics/controller_test.go

```diff
@@ -69,7 +70,8 @@ func assertMetricData(t *testing.T, exporter *metricExporter, metricName string,
 		}
 		e, ok := expectedValuesAsMap[serialize(actualLabelValues)]
 		assert.True(t, ok, "no TimeSeries found with labels: %v", actualLabelValues)
-
+		fmt.Println("ACT", actualLabelValues)
```

Comment:
There is no need to print this or we could use `t.Log`

Reply:
It was used for a debug purpose, I will remove it.